docs/app/Components/Sidebar/Sidebar.js

```diff
@@ -29,17 +29,16 @@ export default class Sidebar extends Component {
       })
       .value()
 
-    const subMenu = (
+    return _.isEmpty(items) ? [] : (
```

Comment:
No longer show menu headings for empty sections.  If the search bar filtered out all "Modules", don't show the "Modules" heading in the sidebar.